src/main/java/uk/gov/hmcts/reform/sscs/callback/handlers/DwpUploadResponseHandler.java

```diff
@@ -47,13 +48,44 @@ public void handle(CallbackType callbackType, Callback<SscsCaseData> callback) {
             throw new IllegalStateException("Cannot handle callback");
         }
 
-        boolean dwpFurtherInfo =
+        BenefitType benefitType = callback.getCaseDetails().getCaseData().getAppeal().getBenefitType();
+        log.info("BenefitType" + benefitType);
+
+        if (benefitType.getCode().equals("UC")) {
```

Comment:
Is the benefit code always upper case?   If so, then no changes needed,  but would equals ignore case be safer?

Reply:
invert the check so that it checks on the constant in case getCode() is null or use StringUtils

Reply:
done